xbmc/addons/AddonStatusHandler.cpp

```diff
@@ -102,10 +102,7 @@ void CAddonStatusHandler::Process()
       pDialog->SetHeading(CVariant{heading});
       pDialog->SetLine(1, CVariant{24070});
       pDialog->SetLine(2, CVariant{24073});
-
-      //send message and wait for user input
-      ThreadMessage tMsg = {TMSG_DIALOG_DOMODAL, WINDOW_DIALOG_YES_NO, g_windowManager.GetActiveWindow()};
-      CApplicationMessenger::Get().SendMessage(tMsg, true);
+      pDialog->Open();
```

Comment:
The changes in this file look like a change in behaviour or not? Both are blocking calls but the old code was opening the dialog in the main thread and the new code opens it in the current thread.